openmrs/apps/clinical/app.json

```diff
@@ -150,6 +150,16 @@
             "debounceSearch": false,
             "fetchDelay": 2000,
             "serializeSearch": false
-        }
+        },
```

Comment:
How will these be translated ever? we should ideally create keys and then use translation feature

Reply:
This is tricky. Seems like this is "_UI Client configuration_" -- on what CDSS alert options we want to show on screen, and not in the "product". Does this mean, these keys are NOT in code anywhere in Bahmni Clinical codebase? If so, then there can't be any special behavior in code based on options here, since these are custom configuration.

Reply:
As i understand, these keys are being used to decide what "dismissal options" are visible on UI. The config allows users to control this. The actual keys/values are already in Bahmniapps (product codebase), so we are good. thanks!